frontend/sportsmatch-app/src/pages/UserRating.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import {
+  ApiError,
+  RatingControllerService,
+  UserControllerService,
+  UserDTO,
+  UserRatingDTO,
+  UserRatingStatsDTO,
+} from '../generated/api'
+import RatingCard from '../components/RatingCard'
+import UserCard from '../components/UserCard'
+import '../styles/UserRating.css'
+
+export default function UserRating() {
+  const { id } = useParams<{ id: string }>()
+  const userId = parseInt(id!, 10)
+  const [user, setUser] = useState<UserDTO>()
+  const [summary, setSummary] = useState<UserRatingStatsDTO>()
+  const [ratings, setRatings] = useState<UserRatingDTO[]>([])
+  const [page, setPage] = useState<number>(0)
+  const size = 1
```

Comment:
I would set it to the final number at ~20 so that we don't have to go back and change it later. If you want to demo the functionality, use a for loop and add 50 reviews to a user in the data preload section